### PR TITLE
Section border

### PR DESCRIPTION
**Pull request by sentayhuJo**

### CODE REVIEW

File: src/components/Emails/EmailFieldEditor.js [hunk 1]
```diff
@@ -25,23 +26,22 @@ export default class EmailFieldEditor extends Component {
...
 
     const fields = getOrderedHtmlFields({ htmlFields });
     const onSelect = compose(dispatch, emailContentPanelChanged);
-
     return (htmlFields ?
       <div>
         <Accordion
           activeKey={activeField}
-          onSelect={(key) => onSelect(key)}
+          onSelect={(key) => onSelect(key)}s
```
Comment (snozza) [inline src/components/Emails/EmailFieldEditor.js]: What's this 's'?

File: src/components/Emails/EmailForm.js [hunk 2]
```diff
@@ -122,12 +122,12 @@ export class EmailFieldForm extends Component {
                   : null}
                 </Col>
                 <Col sm={12}>
-                  <ButtonToolbar>
-                    <Button className="pull-left" type="submit" bsStyle="primary">&nbsp;
+                  <ButtonToolbar className="button-Tool-bar">
```
Comment (snozza) [inline src/components/Emails/EmailForm.js]: No need for capital in Tool bar

File: src/components/Emails/EmailPanel.css [hunk 3]
```diff
@@ -6,3 +6,6 @@
 .panel-body .alert {
   margin-bottom: 0;
 }
+.button-Tool-bar {
```
Comment (snozza) [inline src/components/Emails/EmailPanel.css]: No need for capital T

File: src/components/Twitter/Twitter.css [hunk 4]
```diff
@@ -0,0 +1,6 @@
+.tweet {
+    font-weight: normal;
+}
+em {
```
Comment (snozza) [inline src/components/Twitter/Twitter.css]: Don't change the styling for all em elements - be more specific

File: src/components/Twitter/Twitter.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  get twitterDetail() {
+    return ({
+      label: 'Twitter & Comments',
+      error: '',
+      text: '.@MESandbu%27s%20Free%20Lunch:%20',
+      referer: 'http://www.ft.com/freelunch',
+      via: 'FT%20@FTEconomics',
+      hashtags: 'ftfreelunch'
+    });
+  }
```
Comment (snozza) [inline src/components/Twitter/Twitter.js]: Put a newline between class methods

File: src/components/Twitter/Twitter.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  validateURL(textval) {
+    const urlregex = /^(ftp|http|https):\/\/[^ "]+$/;
+    return urlregex.test(textval);
+  }
```
Comment (snozza) [inline src/components/Twitter/Twitter.js]: Put a newline between class methods

File: src/components/Twitter/Twitter.js [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+  handleGuestChange(field, url = {}) {
+    const { onChange } = this.props;
+    const tweet = {};
+    const newValue = { ...field.url, ...url, ...this.twitterDetail };
+    if (!this.validateURL(url.sharedURL.trim())) {
+      newValue.error = 'Missing or invalid article URL, it should start with http(s).';
+    }
+    tweet.htmlBody = renderTemplate(twitterTemplate, newValue);
+    onChange(tweet);
+  }
```
Comment (snozza) [inline src/components/Twitter/Twitter.js]: Put a newline between class methods

File: src/components/Twitter/Twitter.js [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+import React, { Component } from 'react';
+import { FormGroup, InputGroup, FormControl, ControlLabel, Col } from 'react-bootstrap';
+import renderTemplate from 'helpers/renderTemplate';
+
+import twitterTemplate from 'components/Emails/templates/twitter.html';
+import './Twitter.css';
+
+export default class Twitter extends Component {
+
+  get twitterDetail() {
```
Comment (snozza) [inline src/components/Twitter/Twitter.js]: Can we find a way to hard-wire this into the creation of the component in email-campaigns-api?

File: src/components/Emails/EmailFieldEditor.js (hunk 1 above)
Comment (sentayhuJo) [reply src/components/Emails/EmailFieldEditor.js]: Its a bug, already fix.

File: src/components/Twitter/Twitter.js (hunk 7 above)
Comment (sentayhuJo) [reply src/components/Twitter/Twitter.js]: Done

File: src/components/Twitter/Twitter.js (hunk 6 above)
Comment (sentayhuJo) [reply src/components/Twitter/Twitter.js]: Done

File: src/components/Twitter/Twitter.js (hunk 5 above)
Comment (sentayhuJo) [reply src/components/Twitter/Twitter.js]: Done

File: src/components/Twitter/Twitter.css (hunk 4 above)
Comment (sentayhuJo) [reply src/components/Twitter/Twitter.css]: Changed to class rather using element tag.

File: src/components/Emails/EmailPanel.css (hunk 3 above)
Comment (sentayhuJo) [reply src/components/Emails/EmailPanel.css]: Done

**Pull request merged**